cpp/tests/groupby/groupby_test_util.cpp

```diff
@@ -17,112 +17,125 @@
 #include <cudf/table/table.hpp>
 #include <cudf/types.hpp>
 
-void test_single_agg(cudf::column_view const& keys,
-                     cudf::column_view const& values,
-                     cudf::column_view const& expect_keys,
-                     cudf::column_view const& expect_vals,
-                     std::unique_ptr<cudf::groupby_aggregation>&& agg,
+namespace cudf ::test {
```

Comment:
I disagree with this implementation.
I don't think the test code should be using namespace cudf in this way.

Reply:
Sure, reverted.